src/mlpack/tests/ann_layer_test.cpp

```diff
@@ -5664,6 +5718,91 @@ TEST_CASE("NoBiasConvolutionLayerTestCase", "[ANNLayerTest]")
   REQUIRE(arma::accu(output) == 4108);
 }
 
+/**
+ * Simple test for the Grouped Convolution layer.
```

Comment:
I think this one only uses one group?  It would be worth adding that as a comment here---since C++ doesn't have keyword arguments (yet?), it can be a little tricky to tell these things just from the code.

Reply:
Nope. It uses 2 groups.